ens/async_ens.py

```diff
@@ -118,6 +118,64 @@ class AsyncENS(BaseENS):
     like: ``"0x314159265dD8dbb310642f98f50C066173C1259b"``
     """
 
+    async def address(self, name: str) -> Optional[ChecksumAddress]:
```

Comment:
This was a bit to make these models homogeneous in the order of methods for better code readability:

i.e.
address, setup_address, name, setup_name ... etc ... _get_resolver, _set_resolver (internal methods near the end)

And line up ENS and AsyncENS to be ordered the same way.